test-helpers/src/connection/cassandra.rs

```diff
@@ -179,6 +179,7 @@ impl CassandraConnection {
                     )
                     .user("cassandra", "cassandra")
                     .default_consistency(Consistency::One)
+                    .compression(Some(scylla::transport::Compression::Snappy))
```

Comment:
I'm confused how this works?
Does the driver take this as a suggestion only and drop down to no compression when the DB doesnt support compression?

Reply:
Yes, if the cluster does not support any compression options the driver will default to none.

Reply:
sounds good!